### DISCUSSION TITLE
How do I get discharge_daily.tss instead of discharge_daily.csv

### DISCUSSION THREAD

**Discussion by PengweiZhang1997** [iiasa/CWatM#143] [Q&A] [answered]
I set **OUTPUT OUT_TSS_Daily = discharge** in the settings file and get **discharge_daily.csv** under the PathOut path. However, when I run the calibration, calibration_single.py displays **"No simulated streamflow found"** because my run_calibration\00_id folder only contains **discharge_daily.csv** and not **discharge_daily.tss**. Where can I make changes to get the results in the discharge_daily.tss format?

**Comment by mikhailsmilovic** [comment, marked as answer]
Guten Tag, great question. A recent update has changed the standard text output format to CSV. 
However, for calibration, the script still uses the previous format.

In the CWatM settings template, at the top under [OPTIONS ], include the line:  
### reportOldTss = True

**Reply by mahdimohammadnezhad** [reply]
Hello Professor @mikhailsmilovic,
I tried this for my study area calibration (with reportOldTss = True) but it displayed the same error.
![Screenshot 2025-03-01 170733](https://github.com/user-attachments/assets/b938a349-3899-46b5-abc9-8aaff047ce86)

**Reply by mikhailsmilovic** [reply]
This question is being discussed here: https://github.com/iiasa/CWatM/discussions/183
